_posts/2021-02-02-sc20-report.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: RSE-HPC-2020 Workshop Report
+tags: [conference]
+posted_by: Charles Ferenbaugh, Daniel S. Katz, Sandra Gesing, and Simon Hettrick
+---
+
+We were excited to hold the inaugural [Research Software Engineers in HPC
+Workshop (RSE-HPC-2020)](https://us-rse.org/rse-hpc-2020/) as part of the
+[SC20](https://sc20.supercomputing.org/) conference this year! The half-day,
+virtual event featured 18 speakers and over 100 participants.
+
+The opening session, chaired by Charles Ferenbaugh, began with keynote addresses
+from Simon Hettrick in the UK and Frank Löffler in Germany. They spoke on the
+growth of the RSE movements in their respective countries. Though the name
+"Research Software Engineer" is only about eight years old, the role is much
+older. In those countries (and many others), when RSE events were first
+organized, many RSEs joined in right away and communities grew quickly. Many who
+joined said they were excited to have "found their tribe."
+
+Following the keynotes, three speakers gave lightning talks on their RSE work in
+various contexts: Miranda Mundt on how research software engineers can support
+scientific software, Emilio Carcamo on the Research Software Engineering
+ecosystem for scientific computing supported by SC3UIS in Colombia, and
+Bernadette Boscoe on RSEs as maintainers and their role in keeping code alive.
+
+We continued with two panel discussions; the first of these was on the topic of
+building RSE teams and groups, chaired by Daniel S. Katz with Reed Milewicz,
+Jeremy Cohen, Robert Sinkovits, Mahmood Shad, and Ben van Werkhoven.
+Several panelists described RSE groups that have developed at their
+institutions. In many cases the groups were informal at first, with
+software developers coming together to interact across different
+domains; later they grew into more formal groups with management
+agreeing to provide support. Team-building within groups is an
+important factor, happening both by technical means such as holding
+group meetings and sharing knowledge, and by non-technical means such
+as food and social events. Funding for groups is often a challenge.
+Many groups run primarily on soft money, through short- and medium-term
+projects; this can be effective at a large institution with many small
+projects in flight, but carries more risk at smaller institutions. All
+of the panelists were optimistic about the interesting work available
+to RSEs, and their growing role in the research community.
+
+The second panel, chaired by Sandra Gesing with Joanna Leng, Ian Cosden, Rinku
+Gupta, and Carlos Maltzahn, discussed how to support RSE careers. Some
+institutions have started to recognize RSE roles, but that recognition is uneven
+across the research landscape. Many RSEs in academia and national labs leave for
+industry; this makes retention a challenge, particularly since universities can
+never compete on salary alone. Universities can, however, offer intangibles such
```

Comment:
This next sentence also suggests that national labs aren't part of this set - retention and salary doesn't seem to be a problem there.

Reply:
Resolving here per discussion above.

Reply:
True, but I couldn't think of a way to phrase it that wasn't extremely awkward.